processing/src/main/java/io/druid/query/Druids.java

```diff
@@ -104,9 +120,9 @@ public static AndDimFilterBuilder newAndDimFilterBuilder()
 
   /**
    * A Builder for OrDimFilter.
-   * 
+   * <p/>
```

Comment:
I would prefer that we don't change formatting on the entire file, when doing only localized change, but that's just my personal preference. It makes code reviews harder to try to weed out formatting changes from code changes.